__tests__/app.test.js

```diff
@@ -136,3 +134,52 @@ describe("GET /api documentation", () => {
     })
 
 })
+
+describe("GET /api/articles/:article_id", () => {
+    it("should return a 200 - status when passed with a valid value for the id", () => {
+        return request(app).get("/api/articles/1").expect(200);
+    })
+
+    it("should return a 200 - status and the response body with the required properties", () => {
+        return request(app).get("/api/articles/1").expect(200).then( res => {
+            const { body } = res;
+            const { article } = body;
+
+            expect(article).toHaveProperty("article_id");
+            expect(article).toHaveProperty("author");
+            expect(article).toHaveProperty("title");
+            expect(article).toHaveProperty("body");
+            expect(article).toHaveProperty("topic");
+            expect(article).toHaveProperty("created_at");
+            expect(article).toHaveProperty("votes");
+            expect(article).toHaveProperty("article_img_url");
+        })
+    })
+
+    it("should return the expected values for the article properties as specified in the database", () => {
+        return request(app).get("/api/articles/1").expect(200).then( res => {
+            const { article } = res.body;
+            expect(article.article_id).toBe(1);
+            expect(article.author).toBe('butter_bridge');
+            expect(article.title).toBe('Living in the shadow of a great man');
+            expect(article.body).toBe('I find this existence challenging');
+            expect(article.topic).toBe('mitch');
+            expect(article.created_at).toBe("2020-07-09T20:11:00.000Z");
```

Comment:
Note that should you later make any changes to this article, some of the tests may fail. It may be worth just checking the article_id is 1 and that the others are the expected types (rather than toBe)

Reply:
Wouldn't the reseeding mean it would only edit after this test? @HannahBurks